cmd/ocsp-updater/main.go

```diff
@@ -228,12 +237,14 @@ func (updater *OCSPUpdater) findStaleOCSPResponses(oldestLastUpdatedTime time.Ti
 			 FROM certificateStatus AS cs
 			 JOIN certificates AS cert
 			 ON cs.serial = cert.serial
-			 WHERE cs.ocspLastUpdated < :lastUpdate
+			 WHERE cs.ocspLastUpdated > :maxAge
```

Comment:
@rolandshoemaker for the "real world" those would get caught by the `newCertificatesTick` and the `getCertificatesWithMissingResponses` query.

The problem was just this unit test''s expectation that calling `findStaleOCSPResponses` with test rows it added itself would work.

Make sense?